be/src/Infrastructure/Localization/Foundation.Localization/Resources/ProductResources.pl.resx

```diff
@@ -162,6 +162,18 @@
   <data name="IsPublished" xml:space="preserve">
     <value>Opublikowany</value>
   </data>
+  <data name="NavigateToAttributesLabel" xml:space="preserve">
+    <value>Powróć do atrybutów</value>
+  </data>
+  <data name="NavigateToCategoriesLabel" xml:space="preserve">
+    <value>Powróć do kategorii</value>
+  </data>
+  <data name="NavigateToProductAttributeLabel" xml:space="preserve">
+    <value>Powróć do atrybutu</value>
+  </data>
+  <data name="NavigateToProductsLabel" xml:space="preserve">
+    <value>Powróc do listy produktów</value>
```

Comment:
**suggestion:** Fix typos and keep the convention. It should be "Powróć do produktów"